### PR TITLE
Add paleo compsets

### PR DESCRIPTION
**Pull request by fischer-ncar**
Add 2 new paleo compsets, BmidH and B127ka.  These are a hand me down from MVR.

User interface changes?: No
[ If yes, describe what changed, and steps taken to ensure backward compatibilty ]

Fixes: [Github issue #s] And brief description of each issue.

Testing:  Ran BmidH and B127ka at f09_g17 for one month
  unit tests:
  system tests:
  manual testing:

### CODE REVIEW

**Comment by fischer-ncar** [comment]
Yes, they wanted this in the release.

File: cime_config/usermods_dirs/BmidH/#user_nl_clm# [hunk 1]
```diff
@@ -0,0 +1,107 @@
+!----------------------------------------------------------------------------------
```
Comment (billsacks) [inline cime_config/usermods_dirs/BmidH/#user_nl_clm#]: It looks like this file was accidentally added. It should be removed.

File: cime_config/usermods_dirs/B127ka/user_nl_cism [hunk 2]
```diff
@@ -0,0 +1,20 @@
+!-----------------------------------------------------------------------
```
Comment (billsacks) [inline cime_config/usermods_dirs/B127ka/user_nl_cism]: This effectively empty user_nl file should be removed.

File: cime_config/usermods_dirs/BmidH/#user_nl_clm# (hunk 1 above)
Comment (fischer-ncar) [reply cime_config/usermods_dirs/BmidH/#user_nl_clm#]: Removed

File: cime_config/usermods_dirs/B127ka/user_nl_cism (hunk 2 above)
Comment (fischer-ncar) [reply cime_config/usermods_dirs/B127ka/user_nl_cism]: Removed

**Comment by billsacks** [comment]
@fischer-ncar thanks for your quick start on addressing my requests. I should clarify / correct that I don't feel that the introduction of a shared directory for the SourceMods is essential... I personally think it's a good idea for long-term maintainability, but I'd be okay with letting this PR go through without that. The other comments feel more important.

**Comment by fischer-ncar** [comment]
@billsacks I removed the empty SourceMods, and created a B_paleo_shared for the common SourceMods.

**Pull request merged**